service/tutorial/order/NewTutorialServices.xml

```diff
@@ -0,0 +1,201 @@
+            <parameter name="maritalStatusEnumId"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="personList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.party.Person" list="personList">
+                <econdition field-name="maritalStatusEnumId"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <!-- 3.Finding all Order Header records so that the latest placed order appears at the top-->
+
+    <service verb="get" noun="LatestOrders">
+        <out-parameters>
+            <parameter name="ordersList" type="list"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList">
+                <order-by field-name="placedDate"/>
+            </entity-find>
+        </actions>
+    </service>
+
+    <!-- 4.Finding the Order Header details for a given orderId.-->
+
+    <service verb="get" noun="Order">
+        <in-parameters>
+            <parameter name="orderId" required="true"/>
+        </in-parameters>
+        <out-parameters>
+            <parameter name="orderDetails" type="list"/>
+            <parameter name="orderId"/>
+            <parameter name="orderName"/>
+            <parameter name="placedDate"/>
+            <parameter name="statusId"/>
+            <parameter name="orderRevision"/>
+            <parameter name="currencyUomId"/>
+            <parameter name="productStoreId"/>
+            <parameter name="salesChannelEnumId"/>
+        </out-parameters>
+        <actions>
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList"/>
+            <set field="products" from="[]"/>
+
+            <if condition="placedDate">
+                <then>
+                    <set field="orderId" from="order.orderId"/>
+                    <set field="orderName" from="order.orderName"/>
+                    <set field="placedDate" from="order.placedDate"/>
+                    <set field="statusId" from="order.statusId"/>
+                    <set field="orderRevision" from="order.orderRevision"/>
+                    <set field="currencyUomId" from="order.currencyUomId"/>
+                    <set field="productStoreId" from="order.productStoreId"/>
+                    <set field="salesChannelEnumId" from="order.salesChannelEnumId"/>
+                </then>
+            </if>
+         </actions>
+    </service>
+
+    <!-- 5.Finding all Order Header records with the grandTotal greater than 50.-->
+    <service verb="get" noun="OrderHeaders">
+        <out-parameters>
+            <parameter name="orderCount" type="list"/>
+        </out-parameters>
+        <actions>
+            <set field="orderCount" from="[]"/>
+            <entity-find entity-name="mantle.order.OrderHeader" list="ordersList">
+            </entity-find>
+            <iterate list="ordersList">
+                <if condition="grandTotal &gt; 50">
```

Comment:
Please check the use of operator attribute with entity-find, this condition can be added when performing entity find only.